src/views/Nft/market/Collection/Items/CollectionNfts.tsx

```diff
@@ -45,7 +45,13 @@ const CollectionNfts: React.FC<CollectionNftsProps> = ({ collection, sortBy = 'u
       alignItems="start"
     >
       {orderedNfts.map((nft) => {
-        return <CollectibleLinkCard key={`${nft.tokenId}-${nft.collectionName}`} nft={nft} />
+        return (
+          <CollectibleLinkCard
+            key={`${nft.name}-${nft.collectionName}`}
```

Comment:
In this case these nfts are `PancakeBunnyNftWithTokens`, i.e. different pancake bunnies with tokens. They don't have tokenId (on main level), but name is unique.
We might consider storing bunnyId if we would fetch all attributes, but I'm not sure its needed at this stage. Can be improved later of course, this PR is not "final fix for everything"